src/region/RegionManager.tsx

```diff
@@ -4,31 +4,28 @@ import BaseManager, { Options, Props } from '../common/BaseManager';
 import RegionContainer from './RegionContainer';
 
 export default class RegionManager implements BaseManager {
-    page: number;
+    location: number;
```

Comment:
I'm trying to de-emphasize the concept of pages within the annotator components. We may want to use these on non-paged files eventually.